tasklist/importer-860/src/main/java/io/camunda/tasklist/zeebeimport/v860/processors/os/VariableZeebeRecordProcessorOpenSearch.java

```diff
@@ -36,19 +39,95 @@ public class VariableZeebeRecordProcessorOpenSearch {
 
   @Autowired private TasklistProperties tasklistProperties;
 
+  @Autowired private TasklistListViewTemplate tasklistListViewTemplate;
+
   public void processVariableRecord(final Record record, final List<BulkOperation> operations)
       throws PersistenceException {
     final VariableRecordValueImpl recordValue = (VariableRecordValueImpl) record.getValue();
 
     // update variable
     if (record.getIntent().name() != Intent.MIGRATED.name()) {
       operations.add(persistVariable(record, recordValue));
+      operations.add(persistVariableToListView(record, recordValue));
     }
   }
 
   private BulkOperation persistVariable(
       final Record record, final VariableRecordValueImpl recordValue) throws PersistenceException {
+    final VariableEntity entity = getVariableEntity(record, recordValue);
+    return getVariableQuery(entity);
+  }
+
+  private BulkOperation getVariableQuery(final VariableEntity entity) throws PersistenceException {
+    LOGGER.debug("Variable instance for list view: id {}", entity.getId());
+    final Map<String, Object> updateFields = new HashMap<>();
+    updateFields.put(VariableIndex.VALUE, entity.getValue());
+    updateFields.put(VariableIndex.FULL_VALUE, entity.getFullValue());
+    updateFields.put(VariableIndex.IS_PREVIEW, entity.getIsPreview());
+
+    return new BulkOperation.Builder()
+        .update(
+            up ->
+                up.index(variableIndex.getFullQualifiedName())
+                    .id(entity.getId())
+                    .document(CommonUtils.getJsonObjectFromEntity(entity))
+                    .docAsUpsert(true)
+                    .retryOnConflict(OpenSearchUtil.UPDATE_RETRY_COUNT))
+        .build();
```

Comment:
It seems that `updateFields` is not used in Opensearch 
```suggestion
    return new BulkOperation.Builder()
        .update(
            up ->
                up.index(variableIndex.getFullQualifiedName())
                    .id(entity.getId())
                    .document(CommonUtils.getJsonObjectFromEntity(updateFields))
                    .upsert(CommonUtils.getJsonObjectFromEntity(entity))
                    .retryOnConflict(OpenSearchUtil.UPDATE_RETRY_COUNT))
        .build();
```
same should be applied for `VariableListViewEntity`